synapse/lib/ast.py

```diff
@@ -39,6 +39,7 @@ class AstNode:
     def __init__(self, kids=()):
         self.kids = []
         self.hasast = {}
+        self.lines = (-1, -1)
```

Comment:
Could we make this more granular ( ie, the exact text offsets ) and not break the covaregeio use case?

Reply:
we could add text offsets for our future use as well, but coverage only cares about lines